### PR TITLE
Revert EN JSON changes

### PR DESCRIPTION
**Pull request by hahmadia**
#### Summary

Reverts i18n changes made in #14976

#### Jira Ticket
https://mattermost.atlassian.net/browse/MM-29097

This PR is needed for https://github.com/mattermost/enterprise/pull/783

### CODE REVIEW

File: api4/job_test.go [hunk 1]
```diff
@@ -174,108 +172,113 @@ func TestGetJobsByType(t *testing.T) {
 	CheckForbiddenStatus(t, resp)
 }
 
-func TestDownloadJob(t *testing.T) {
```
Comment (sbishel) [inline api4/job_test.go:177]: Go ahead and remove this rather than just commenting out.

**Review by sbishel — APPROVED** [review]

**Review by mkraft — APPROVED** [review]

**Comment by sbishel** [comment]
@amyblais Yes, I believe @wiersgallak has already started that process.

**Comment by wiersgallak** [comment]
@amyblais I did remove it from the Release announcement already.

**Review by srkgupta — APPROVED** [review]
Tested the changes along with its dependent PR and the issue is fixed now and also did a regression testing around Compliance export and no issues were found. Approving the PR.

**Pull request merged**